foaf.json

```
@@ -0,0 +1,22 @@
+{
+	"@context": {
+		"foaf": "http://xmlns.com/foaf/0.1/",
+		"rdf": "http://www.w3.org/2000/01/rdf-schema#",
```

```suggestion
		"rdf": "http://www.w3.org/2000/01/rdf-schema#"
```
This comma must go 💣

---

👍

---

noob